x-pack/plugins/apm/public/components/shared/Stacktrace/Variables.tsx

```diff
@@ -27,49 +27,34 @@ const VariablesContainer = styled.div`
   font-family: ${fontFamily};
```

Comment:
This should be removed as it's causing the font to render as Open Sans instead of the default Kibana font, Inter UI.

Reply:
That's right. Font-family removed.